tsdb/head.go

```diff
@@ -447,6 +448,13 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 				60 * 60 * 12, // 12h
 			},
 		}),
+		headChunksStorageSize: prometheus.NewGaugeFunc(prometheus.GaugeOpts{
+			Name: "prometheus_tsdb_head_chunks_storage_size_bytes",
+			Help: "Size of the chunks_head directory.",
+		}, func() float64 {
+			val, _ := h.chunkDiskMapper.Size()
```

Comment:
This solution is similar to this implemented one https://github.com/infracloudio/prometheus-oss/blob/32ed6101b811fcf09bff4aab14166692069ebfa9/tsdb/head.go#L2025

It can encounter error, even if we encounter we should be sending only values from this function.
Other solution we can do is log the error.

Let me know your thoughts.

Reply:
lgtm